ios/PurchasesHybridCommon/PurchasesHybridCommonTests/PurchasesHybridCommonTests.swift

```diff
@@ -361,5 +361,20 @@ class PurchasesHybridCommonTests: QuickSpec {
                 }
             }
         }
+
+        context("setLogHandler") {
+            let expectedMessage = "a message"
+            for level in LogLevel.levels {
+                it("\(level) logs work") {
+                    let originalHandler = Purchases.verboseLogHandler
+                    CommonFunctionality.setLogHander(completion: { logDetails in
+                        expect(logDetails["logLevel"]) == level.description.uppercased()
```

Comment:
I don't think you need `uppercased` here since it's already uppercase.

Reply:
oh yes! I didn't notice

Reply:
actually, I think it would be good to leave it just in case the description changes in the future to lowercase. I don't think it will but it would break the dart implementation if it does :)

Reply:
Ok that makes sense